### PR TITLE
[VEN-2192]: Proposal for VTreasury opBNB Testnet

### PR DESCRIPTION
**Pull request by GitGuru7**
Resolves VEN-2192

### CODE REVIEW

File: src/networkAddresses.ts [hunk 1]
```diff
@@ -69,4 +69,8 @@ export const NETWORK_ADDRESSES = {
     XVS_STORE: sepolia_deployed_contracts.contracts.XVSStore.address,
     GENERIC_TEST_USER_ACCOUNT: "0x2Ce1d0ffD7E869D9DF33e28552b12DdDed326706",
   },
+  opbnbtestnet: {
+    NORMAL_TIMELOCK: "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf",
+    VTREASURY: "0x3370915301E8a6A6baAe6f461af703e2498409F3",
```
Comment (chechu) [inline src/networkAddresses.ts:74]: I wouldn't add this address to this file. I would hardcode it in the VIP and simulation files. Only addresses used by the framework should be defined in this file

File: multisig/proposals/vip-005/vip-005-opbnbtestnet.ts
Comment (chechu) [inline multisig/proposals/vip-005/vip-005-opbnbtestnet.ts:1]: What do you think if we move this VIP to `/multisig/proposals/vip-000/vip-000-opbnbtestnet.ts`?

I think we'll have similar VIPs for the setup of the networks, and keeping together the VIP's doing exactly the same things could help us to reduce the disorder :)

Comment (GitGuru7) [reply multisig/proposals/vip-005/vip-005-opbnbtestnet.ts:1]: [613f383](https://github.com/VenusProtocol/vips/pull/116/commits/613f383e331745402e1ff9658f3740c4fa51e713)

**Review by chechu — APPROVED** [review]

**Comment by GitGuru7** [comment]
https://testnet.opbnbscan.com/tx/0x879591f61fe80bee25cd56ab6e1474be20bd00d6de5dec55bba829aabab2000f?tab=overview

**Pull request merged**